File: scratch/list_zip_contents.js
const { execSync } = require("child_process");

function run() {
  console.log("Listing contents of C:\\Users\\LENOVO\\Downloads\\ibps clerk.zip using PowerShell...");
  try {
    const cmd = `powershell -Command "[System.Reflection.Assembly]::LoadWithPartialName('System.IO.Compression.FileSystem') ; [System.IO.Compression.ZipFile]::OpenRead('C:\\Users\\LENOVO\\Downloads\\ibps clerk.zip').Entries | Select-Object -Property FullName, Length"`;
    const output = execSync(cmd, { encoding: "utf8" });
    console.log(output);
  } catch (err) {
    console.error("Error listing zip contents:", err.message);
  }
}

run();

File: scratch/list_zip_contents.py
import zipfile
import os

zip_path = "C:\\Users\\LENOVO\\Downloads\\ibps clerk.zip"
if not os.path.exists(zip_path):
    print("Zip file not found at", zip_path)
else:
    print("Listing zip contents for:", zip_path)
    with zipfile.ZipFile(zip_path, 'r') as zip_ref:
        for info in zip_ref.infolist():
            print(f"  {info.filename} ({info.file_size} bytes)")
